horovod/common/operations.cc

```diff
@@ -75,6 +108,63 @@ namespace horovod {
 namespace common {
 
 namespace {
+    
+#if HAVE_MLSL
+#define COMMUNICATOR_TABLE_MAX 1000
+#define NULL_VAL nullptr
```

Comment:
Let's just use `nullptr` literal everywhere instead of a macro.

Reply:
Done